src/transport_performance/gtfs/gtfs_utils.py

```diff
@@ -7,20 +7,135 @@
 import os
 import math
 import plotly.graph_objects as go
-from typing import Union
+from typing import Union, TYPE_CHECKING
 import pathlib
 from geopandas import GeoDataFrame
+import warnings
+
+if TYPE_CHECKING:
+    from transport_performance.gtfs.validation import GtfsInstance
 
 from transport_performance.utils.defence import (
     _is_expected_filetype,
     _check_iterable,
     _type_defence,
     _validate_datestring,
     _enforce_file_extension,
+    _gtfs_defence,
 )
 from transport_performance.utils.constants import PKG_PATH
 
 
+def _validate_bbox(bbox: list) -> None:
```

Comment:
`_validate_bbox()` -> `_validate_bbox_list()` I'm guessing the other devs who tend to work with polygons may get distracted by the name.

Reply:
Good shout.